Source/WTF/wtf/text/StringView.h

```diff
@@ -1033,7 +1044,7 @@ inline auto StringView::CodeUnits::Iterator::operator++() -> Iterator&
 
 inline UChar StringView::CodeUnits::Iterator::operator*() const
 {
-    return m_stringView[m_index];
+    return m_stringView.unsafeCharacterAt(m_index);
 }
```

Comment:
I understand why this is needed for speed, but not why it’s intrinsically safe. It seems clients of the iterator remain unsafe.

Reply:
Yes, this is not perfect. I can try removing this in a follow-up after perf testing this particular change. I was happy to finally get a patch that didn't regress benchmarks so I uploaded it for review.

While this isn't fully safe, this is usually used in `for` range loops and thus doesn't end up being super risky.